packages/app/obojobo-express/webpack.config.js

```diff
@@ -44,6 +45,7 @@ module.exports =
 				watchOptions: {
 					ignored: '/node_modules/'
 				},
+				setup: ObojoboExpressDev,
```

Comment:
The setup option is deprecated. Suggest moving into `before` function.

Reply:
#busted